internal/provider/resource_gitlab_project_protected_environment.go

```diff
@@ -0,0 +1,198 @@
+package provider
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+// https://docs.gitlab.com/ee/ci/environments/protected_environments.html
+var _ = registerResource("gitlab_project_protected_environment", func() *schema.Resource {
+	// lintignore: XR002 // TODO: Resolve this tfproviderlint issue
+	return &schema.Resource{
+		Description: "This resource allows you to create and manage Project Access Token for your GitLab projects.",
```

Comment:
Same thing I already mentioned in the other resource with the description consistency ...